src/Home.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from "react";
+import { styled } from "@mui/material/styles";
+import Card from "@mui/material/Card";
+import CardHeader from "@mui/material/CardHeader";
+import CardMedia from "@mui/material/CardMedia";
+import CardContent from "@mui/material/CardContent";
+import CardActions from "@mui/material/CardActions";
+import Collapse from "@mui/material/Collapse";
+import Avatar from "@mui/material/Avatar";
+import IconButton, { IconButtonProps } from "@mui/material/IconButton";
+import Typography from "@mui/material/Typography";
+import { red } from "@mui/material/colors";
+import FavoriteIcon from "@mui/icons-material/Favorite";
+import ShareIcon from "@mui/icons-material/Share";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import MoreVertIcon from "@mui/icons-material/MoreVert";
+import { Container, Grid } from "@mui/material";
+import { ButtonWrapper, CardWrapper, AppContainer } from "./globalStyles";
+import { dummyData } from "./utils/constant";
+import { dummyDataProps } from "./utils/types";
+
+interface ExpandMoreProps extends IconButtonProps {
+  expand: boolean;
+}
+
+export default function Home() {
+  return (
+    <AppContainer maxWidth="lg" sx={{ margin: "40px" }}>
+      <Grid container spacing={5}>
+        {dummyData.map((item: dummyDataProps) => (
+          <Grid item xs={4} md={4}>
```

Comment:
You need to decrease the value of "xs" to 2 or 1 because right now the this container is not responsive at all.